models/quoteCollection.js

```diff
@@ -7,10 +7,15 @@ const { ObjectId } = Schema;
 
 const quoteCollection = new Schema({
   owner: ObjectId,
-  title: String,
-  description: String,
+  title: { type: String, required: true, maxlength: 140 },
+  description: { type: String, maxlength: 500 },
```

Comment:
Be consistent about formatting your schemas. I would stick with multiline everywhere... :bowtie: